www/include/configuration/configKnowledge/display-hosts.php

```diff
@@ -112,24 +119,30 @@
     $query .= "WHERE host.host_id = ehi.host_host_id ";
     if ($postPoller !== false) {
         $query .= "AND nhr.nagios_server_id = :postPoller ";
+        $queryValues[':postPoller'] = [
+            PDO::PARAM_INT => $postPoller
+        ];
     }
     $query .= "AND host.host_register = '1' ";
     if ($postHostgroup !== false) {
         $query .= "AND hgr.hostgroup_hg_id = :postHostgroup ";
+        $queryValues[':postHostgroup'] = [
+            PDO::PARAM_INT => $postHostgroup
+        ];
     }
     if (!empty($postHost)) {
         $query .= "AND host_name LIKE :postHost ";
+        $queryValues[':postHost'] = [
+            PDO::PARAM_STR => "%" . $postHost . "%"
+        ];
     }
-    $query .= "ORDER BY " . $orderby . " " . $order . " LIMIT " . $num * $limit . ", " . $limit;
+    $query .= "ORDER BY " . $orderBy . " " . $order . " LIMIT " . $num * $limit . ", " . $limit;
```

Comment:
could you bind offset and limit like you did here : https://github.com/centreon/centreon/pull/8815/commits/c812cb04e0454e113c5dde7368798265f11d7cc1#diff-e8d3d31415f23da2771e17637d4355f9R113